README.md

```diff
@@ -1,86 +1,146 @@
-- [Mozilla hosts a community effort called `libdweb`](https://github.com/mozilla/libdweb/) to implement experimental APIs for Firefox WebExtensions with a goal of enabling dweb protocols in Firefox through browser add-ons. [IPFS libdweb experiments](https://github.com/ipfs-shipyard/ipfs-companion/blob/libdweb/docs/libdweb.md) include [native protocol handler](https://github.com/ipfs-shipyard/ipfs-companion/pull/533), [local DNS-SD discovery and TCP transport](https://github.com/ipfs-shipyard/ipfs-companion/pull/553). The long term goal of this project is to integrate these APIs into the WebExtensions ecosystem.
-- [Support `chrome.sockets.*` APIs in Chromium browsers](https://github.com/ipfs-shipyard/ipfs-companion/issues/664)
+#### Other web extension experiments
+- Exposing the IPFS API via [`window.ipfs`](https://docs.ipfs.io/how-to/companion-window-ipfs/) 
+- Mozilla hosted a community effort called [`libdweb`](https://github.com/mozilla/libdweb/) to implement experimental APIs for Firefox WebExtensions, with a goal of enabling dweb protocols in Firefox through browser add-ons:
+  - IPFS [libdweb experiments](https://github.com/ipfs-shipyard/ipfs-companion/blob/libdweb/docs/libdweb.md), including a [native protocol handler](https://github.com/ipfs-shipyard/ipfs-companion/pull/533), [local DNS-SD discovery and TCP transport](https://github.com/ipfs-shipyard/ipfs-companion/pull/553)
+  - The long-term goal of this project was to integrate these APIs into the WebExtensions ecosystem, but it did not get into Firefox Nightly yet (Q3 2020).
+- Support for [`chrome.sockets.*` APIs](https://github.com/ipfs-shipyard/ipfs-companion/issues/664) in Chromium browsers, deprioritized due to [EOL 2022](https://9to5google.com/2020/01/15/google-killing-chrome-apps/)
+
+### IPFS and the JavaScript ecosystem
+At present, in order to run IPFS in a web browser, you must either bundle [`js-ipfs`](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs) (a full IPFS node in JavaScript) with your client-side application, or use the [`js-ipfs-http-client`](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs-http-client) HTTP API client library to connect to an external daemon running on a local or remote machine. To learn more, make sure to check [`js-ipfs/examples`](https://github.com/ipfs/js-ipfs/tree/master/examples) and [`ipfs-http-client/examples`](https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs-http-client/examples).
 
-### JavaScript Ecosystem
-Currently in order to run IPFS in a web browser, you have to either bundle [`js-ipfs`](https://github.com/ipfs/js-ipfs) (**full IPFS node in JS**) with your client-side application
-or use [`js-ipfs-http-client`](https://github.com/ipfs/js-ipfs-http-client) (**HTTP API client library**) to connect to external daemon running on local or remote machine. Make sure to check `/examples` in both repos.
+#### ... in shared worker 
```

Comment:
@Gozala I've  added your work here, but I think we could move node sharing into separate section and include cross origin and leveraging Desktop. 
Don't want to block this PR, so feel free to expand in separate PR.